app.py

```diff
@@ -103,13 +107,20 @@ def create_app():
     else:
         os.environ['FLASK_ENV'] = 'production'
         application.config.from_object('config.ProdConfig')
-
+    application.config['SWAGGER'] = {
+        'title': 'Delivery System API',
+        'description': 'Documentation for Delivery App by Varvara M.',
+        'doc_dir': './apidocs/'
+    }
+    Swagger(application, parse=True)
```

Comment:
why is `parse=True` needed?

also, FYI: swagger config can be passed directly to Swagger object: `Swagger(app, swagger_config)`

Reply:
"parse=True" was delted in last commit. It is not needed in my case